src/render/painter.js

```diff
@@ -266,6 +266,14 @@ class Painter {
         return new DepthMode(func || this.context.gl.LEQUAL, mask, [nearDepth, farDepth]);
     }
 
+    listImages() {
```

Comment:
Although we do technically attach the style's `imageManager` instance to the `Painter`, it's more of a member of the `Style` class, which also manages other `ImageManager`-related APIs (`addImage`/`getImage`/`removeImage`) — let's move this there.